packages/grafana-prometheus/src/language_provider.ts

```diff
@@ -251,6 +251,7 @@ export default class PromQlLanguageProvider extends LanguageProvider {
     const res = await this.request(url, [], searchParams, this.getDefaultCacheHeaders());
     if (Array.isArray(res)) {
       this.labelKeys = res.slice().sort();
+      return res.slice().sort();
```

Comment:
why do we sort twice? We've already sorted the keys above. Can't we just return it?

Reply:
Wanted to prevent race conditions, but makes sense. https://github.com/grafana/grafana/pull/102318/